src/query/src/sql.rs

```diff
@@ -93,3 +106,124 @@ pub fn show_tables(stmt: ShowTables, catalog_manager: CatalogManagerRef) -> Resu
         .context(error::CreateRecordBatchSnafu)?;
     Ok(Output::RecordBatches(records))
 }
+
+pub fn describe_table(stmt: DescribeTable, catalog_manager: CatalogManagerRef) -> Result<Output> {
```

Comment:
Could you please cover this method in unit test? You can use [`MemoryCatalogManager`](https://github.com/GreptimeTeam/greptimedb/blob/7ba512980a89b76f333af34fa55099f2ddab4d13/src/catalog/src/local/memory.rs#L66) for test like [`this`](https://github.com/GreptimeTeam/greptimedb/blob/7ba512980a89b76f333af34fa55099f2ddab4d13/src/servers/tests/mod.rs#L90)

Reply:
I've added some tests, I think they cover most of cases. Please give a look